src/main/java/Data/User.java

```diff
@@ -0,0 +1,63 @@
+package Data;
+
+public class User {
+
+    String username;
+    String password;
+    String email;
+    String userID;
+
+    public User(String username, String password, String email, String userID) {
+        this.username = username;
+        this.password = password;
+        this.email = email;
+        this.userID = userID;
+    }
+
+    public User (String line) {
+        String[] parts = line.split(",");
+        if (parts.length == 4) {
```

Comment:
i think this validation here is not a good place. File validation belongs to User Manager

Reply:
Not done.